jujugui/templates/index.html.go

```diff
@@ -179,15 +179,71 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
       var flags = {}; // Declare an empty set of feature flags.
 
+      class MessageRotator {
```

Comment:
Can you add a comment to each file saying that any updates also need to be applied to the other.